docs/preview/yamlgettingstarted-triggers.md

```diff
@@ -32,8 +34,41 @@ trigger:
 
 Note, path filters are only supported for Git repositories in VSTS.
 
-## CI is opt-out
+### CI is opt-out
 
 Continuous integration builds can be turned off by specifying `trigger: none`
 
 Optionally, the triggers can be managed from the web definition editor, on the Triggers tab.
+
+## Pull request triggers (currently for GitHub repositories only)
+
+Pull request builds are on by default for all branches. Optionally an explicit list of branch filters can
+be specified in the YAML file.
+
+When a PR is updated, the YAML file from the target branch is used to evaluate the branch filters.
```

Comment:
Why would the target branch be used? If I remove or add a PR trigger in my PR because we are removing a configuration that needs building, I want to reflect that in the PR that is making the change.

Reply:
(This, btw, is one of the biggest shortcomings of .NET CI today. Jobs must be 'committed' prior to taking effect).

Reply:
That seems to be what you want because if I can remove the target I can now unprotect master from my PR with out any action from the repo owner

Reply:
Yea what you are saying is probably fine. I am trying to think through scenarios again (including fork scenarios), where this concern would matter. I think you are right, it doesn't matter.

Reply:
@chrisrpatterson It is the repo owner's job to review the code in the PR.  The CI file cannot protect someone from doing malicious things, it just moves it downstream.  If I can't change the CI file, i'll just change build.cmd, msbuild project, etc. to exit immediately.

Reply:
@mmitche what about more policy oriented things.  For example there is an option on the PR trigger to enable secrets to be used during a PR build.  I don't think we should let you change that and have it take affect as part of the PR.

Reply:
@chrisrpatterson Yeah, we wouldn't want that behavior.  Though I could see for a non-OSS runs that wouldn't be an issue.  So maybe a base policy is made at the project/build def level?

Not sure